code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -304,6 +304,10 @@
 
 	var/is_zoomed = 0
 	var/zoom_turf = null
+
+	///Type of weeds the xeno is standing on, null when not on weeds
+	var/obj/effect/alien/weeds/loc_weeds_type = null
```

Comment:
```suggestion
	///Type of weeds the xeno is standing on, null when not on weeds
	var/obj/effect/alien/weeds/loc_weeds_type
```
Also is this a typepath or a reference, and if its a path why is it cast

Reply:
The typepath is needed when doing `initial(loc_weeds_type.<weed_property>)`